cmd/copyTraverser.go

```diff
@@ -23,55 +23,47 @@ package cmd
 import (
 	"context"
 	"errors"
+	"fmt"
+	"github.com/Azure/azure-storage-azcopy/common"
 	"github.com/Azure/azure-storage-azcopy/ste"
 	"net/url"
 	"strings"
 )
 
-func newBlobTraverserForRemove(cca *cookedCopyCmdArgs) (t *blobTraverser, err error) {
+// TODO implement for local and ADLS Gen2
+func newTraverserForCopy(targetURL string, targetSAS string, targetType common.Location,
```

Comment:
This is better than before, but we should REALLY have a global newTraverser function.

I already have this in the refactor. Perhaps my PR after the refactor will be cleaning all of these up.

Reply:
ok!